frontend/src/scenes/surveys/SurveyResponseLimitWidget.tsx

```diff
@@ -0,0 +1,64 @@
+import { IconInfo } from '@posthog/icons'
+import { LemonBanner, Link, Tooltip } from '@posthog/lemon-ui'
+import { useValues } from 'kea'
+import { billingLogic } from 'scenes/billing/billingLogic'
+import { userLogic } from 'scenes/userLogic'
+
+const DEFAULT_SURVEY_RESPONSE_LIMIT = 250
+
+export function SurveyResponseLimitWidget(): JSX.Element | null {
+    const { billing } = useValues(billingLogic)
+    const { user } = useValues(userLogic)
+
+    // Only show for non-admin users
+    if (user?.is_staff || user?.is_impersonated) {
```

Comment:
`is_staff` and `is_impersonated` are not for user roles, but instead, Django attributes to determine wether the current user has access to the admin panel or if they are impersonating someone

for checking if they are an admin, you can instead import the organization logic like this:

`const { isAdminOrOwner } = useValues(organizationLogic)`

there are some examples in the codebase